crypto/kyber/pqcrystals_kyber_ref_common/kem.c

```diff
@@ -23,18 +51,15 @@
 int crypto_kem_keypair(uint8_t *pk,
                        uint8_t *sk)
 {
-  size_t i;
-  indcpa_keypair(pk, sk);
-  for(i=0;i<KYBER_INDCPA_PUBLICKEYBYTES;i++)
-    sk[i+KYBER_INDCPA_SECRETKEYBYTES] = pk[i];
-  hash_h(sk+KYBER_SECRETKEYBYTES-2*KYBER_SYMBYTES, pk, KYBER_PUBLICKEYBYTES);
-  /* Value z for pseudo-random output on reject */
-  pq_custom_randombytes(sk+KYBER_SECRETKEYBYTES-KYBER_SYMBYTES, KYBER_SYMBYTES);
+  uint8_t coins[2*KYBER_SYMBYTES];
+  pq_custom_randombytes(coins, KYBER_SYMBYTES);
+  pq_custom_randombytes(coins+KYBER_SYMBYTES, KYBER_SYMBYTES);
```

Comment:
Is there a need for two calls to the DRBG?
```suggestion
  pq_custom_randombytes(coins, KYBER_SYMBYTES * 2);
```